docs/source/transform.md

```diff
@@ -1,6 +1,20 @@
 Transform
 =========
 
+1. [Introduction](#introduction)
+
+2. [Transform support list](#transform-support-list)
+
+    2.1 [Tensorflow](#tensorflow)
+
+    2.2 [Pytorch](#pytorch)
+
+    2.3 [MXNet](#mxnet)
+
+    2.4 [ONNXRT](#onnxrt)
+
+## Introduction
+
 Neural Compressor supports built-in preprocessing methods on different framework backends. Refer to [this HelloWorld example](/examples/helloworld/tf_example1) on how to configure a transform in a dataloader.
 
 ## Transform support list
```

Comment:
Capitalize the first letter of each word in the title

Reply:
Thanks, fixed.